### DISCUSSION TITLE
Lesson 7, Part 3 - foundry.toml remapping not working

### DISCUSSION THREAD

**Discussion by 0xwtfwtf** [Cyfrin/foundry-full-course-cu#491] [General]
I copied the FundMe.sol and PriceConverter.sol contracts into my VScode, downloaded the brownie contracts and put: remappings = ["@chainlink/contracts/=lib/chainlink-brownie-contracts/contracts/"] into my foundry.toml and tried to compile but kept getting the following error. I was able to sidestep it by going into FundMe.sol and PriceConverter.sol contracts and changing the import line from "@chainlink/contracts/" to "lib/chainlink-brownie-contracts/contracts/"...then compiling worked. So I think it is something wrong with foundry.toml but I don't know what. I did save the changes to foundry.toml before I tried to compile.

Here are the errors I was getting: 

Compiler run failed:
Warning (3420): Source file does not specify required compiler version! Consider adding "pragma solidity ^0.8.19;"
--> src/PriceConverter.sol

Error (2904): Declaration "PriceConverter" not found in "src/PriceConverter.sol" (referenced as "./PriceConverter.sol").
 --> src/FundMe.sol:7:1:
  |
7 | import {PriceConverter} from "./PriceConverter.sol";

**Comment by williamquao** [comment]
For anyone having this problem; 

Find the path AggregatorV3Interface.sol is installed 

```
 find ~/Smart-Contract/foundry-FundMe-f23 -name "AggregatorV3Interface.sol" 


import {AggregatorV3Interface} from "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
import {PriceConverter} from "./PriceConverter.sol";


import {AggregatorV3Interface} from "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
```
